tests/resources/vocabularies/test_subjects_vocabulary.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Subject resource tests."""
+
+import pytest
+from invenio_records_resources.proxies import current_service_registry
+from invenio_vocabularies.contrib.subjects.api import Subject
+
+
+@pytest.fixture(scope="module")
+def subjects_service():
+    return current_service_registry.get("rdm-subjects")
+
+
+@pytest.fixture()
+def example_subject(
+    app, db, es_clear, superuser_identity, subjects_service
+):
+    """Example subject."""
+    data = {
+        "id": "https://id.nlm.nih.gov/mesh/D000001",
+        "scheme": "MeSH",
+        "subject": "Calcimycin",
+    }
+    aff = subjects_service.create(superuser_identity, data)
+    Subject.index.refresh()  # Refresh the index
+
+    return aff
+
+
+@pytest.mark.skip(
```

Comment:
To be investigated https://github.com/inveniosoftware/invenio-vocabularies/issues/66